docs/cna-disclosure-policy.md

```diff
@@ -0,0 +1,63 @@
+# Vulnerability Disclosure Policy
+⚠️  DRAFT
```

Comment:
Feel free to steal the "⚠️  DRAFT" text I've been using in the other documents. I've purposefully added instructions and links to make it easier to contribute in that text. :-)

Reply:
I think I have grabbed the correct form